constants/ourTeam.tsx

```diff
@@ -0,0 +1,91 @@
+import Atharv from '@assets/atharv.png'
+import Aditya from '@assets/aditya.png'
+import Ekta from '@assets/ekta.png'
+import Geo from '@assets/geoffrey.png'
+import Pranshu from '@assets/pranshu.jpeg'
+import Karthik from '@assets/karthik.png'
+import Kushagra from '@assets/kushagra.png'
+import Shrestha from '@assets/shrestha.png'
+import Shubham from '@assets/shubham.png'
+import Sparsh from '@assets/sparsh.png'
+import Tushar from '@assets/tushar.png'
+
+export const values: TeamDataType[] = [
+  {
+    name: 'Atharv Agarwal',
+    image: Atharv,
+    role: 'Secretary',
+    description:
+      'A man with passion and compassion at the same time. Truly a person who can drive multiple forces towards the GOAL. I am sure your abilities will drive Android Club towards a successful and fulfilling journey ahead full of triumph and achievements✨.',
```

Comment:
We do not use lowercase text anywhere in the project. Please convert all descriptions to uppercase. Using a VS Code shortcut for this can be helpful 😄

Reply:
ok